tools/scylla-nodetool.cc

```diff
@@ -89,9 +152,36 @@ const std::map<std::string_view, std::string_view> option_substitutions{
     {"-pw", "--password"},
     {"-pwf", "--password-file"},
     {"-pp", "--print-port"},
+    {"-st", "--start-token"},
+    {"-et", "--end-token"},
 };
 
 const std::map<operation, operation_func> operations_with_func{
+    {{"compact",
+            "Force a (major) compaction on one or more tables",
+R"(
+Forces a (major) compaction on one or more tables. Compaction is an optimization
+that reduces the cost of IO and CPU over time by merging rows in the background.
+
+By default, major compaction runs on all the keyspaces and tables. Major
+compactions will take all the SSTables for a column family and merge them into a
+single SSTable per shard. If a keyspace is provided, the compaction will run on
+all of the tables within that keyspace. If one or more tables are provided as
+command-line arguments, the compaction will run on all tables.
+
+Fore more information, see: https://opensource.docs.scylladb.com/stable/operating-scylla/nodetool-commands/compact.html
+)",
+            {
+                    typed_option<>("split-output,s", "Don't create a single big file (unused)"),
```

Comment:
What does "unused" mean? We don't support this option? If so, why not print an error instead of ignoring it?
Same in all other "unused" options below. Sounds like we need to call them "unimplemented" and print an error if used. Or if you think we should never have these options, just remove them from the list and the user will get an error on an unknown option.

Reply:
I am replicating the behavior of our current nodetool.
You make a good point, I guess now is the time to decide what to do with all these legacy options. I did decide to refuse a legacy option below, because ignoring it made no sense and current nodetool errors out if it is attempted to be used too.
@avikivity asked me to add all these legacy options, I guess this is a continuation of our existing policy of accepting and silently dropping legacy C* options in ScyllaDB, a policy, which is not without controversy.

So, what should we do with these options:
1) Parse and ignore
2) Parse and refuse
3) Not even parse (report unknown option)

Traditionally we have opted for (1), so we don't break any existing script or automation software. One can argue that this is sub-optiomal, the script is possibly still broken because Scylla no longer does what the sript's author wanted, e.g. the output-sstables are silently not split, even when `-s` was passed.